bring-it-up/bring-it-up
# FE#28 Refactor data fetching

## Description
#### Summary:
- Refactors data fetching logic in the frontend so that components don't call `fetch` directly

#### Changes:
- Created new `api.ts` files for functions that fetch data from the server
- Refactored Home and Full Service Page to only call these functions at top-level components

#### How to test this PR:
- Visit the home page and full service pages. All the data should load as expected, and there should be no visible changes to the user
  
## Checklist

- [x] I ran the linter using `npm run lint` and there were no errors
